### PR TITLE
RN-about-eus: updates EUS statement in RNs for 4.14

### PR DESCRIPTION
**Pull request by JoeAldinger**
Updates the EUS statement for 4.14 support

Version(s):
4.14

Issue:

Link to docs preview:
https://63121--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes

QE review:
- [ ] QE has approved this change.

Additional information:
Requesting a merge-review as a double check of updates, please.

### CODE REVIEW

**Comment by nalhadef** [comment]
Hi, Joe. Just a couple of small suggestions.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -25,13 +25,13 @@ You must use {op-system} machines for the control plane, and you can use either
...
+{product-title} 4.14 is an Extended Update Support (EUS) release. More information on Red Hat OpenShift EUS is available in link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[OpenShift Life Cycle] and link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
 
 //TODO: The line below should be used when it is next appropriate. Revisit in August 2023 timeframe.
-//Maintenance support ends for version 4.8 in January 2023 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Maintenance support ends for version 4.12 in January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:34]: January 24, 2025

**Pull request merged**